scripts/word_embeddings/extended_results.ipynb

```diff
@@ -11,8 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook contains the extended results on word embeddings evaluation.\n",
-    "Please see the `word_embeddings_evaluation.ipynb` notebook in the `docs/examples` directory for aggregated results."
+    "This notebook contains the extended results on word embeddings evaluation."
```

Comment:
I moved this notebook to the dmlc/web-data. It just aggregates the .tsv files so that it's easy to compare different embeddings. Thanks for your suggestion about dmlc/web-data